src/OpenTelemetry.Contrib.Instrumentation.Runtime/Implementation/AssembliesInstrumentation.cs

```diff
@@ -0,0 +1,33 @@
+﻿// <copyright file="AssembliesInstrumentation.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Diagnostics.Metrics;
+
+namespace OpenTelemetry.Contrib.Instrumentation.Runtime.Implementation
+{
+    internal class AssembliesInstrumentation : IRuntimeInstrumentation
+    {
+        private readonly Meter meter;
+        private readonly ObservableCounter<int> assemblyCounter;
+
+        public AssembliesInstrumentation(Meter meter)
+        {
+            this.meter = meter;
+            this.assemblyCounter = meter.CreateObservableCounter("assembly-count", () => AppDomain.CurrentDomain.GetAssemblies().Length, description: "Number of Assemblies Loaded");
```

Comment:
I think the names for the metrics in this PR should be prefixed with the `process.runtime.dotnet.` namespace. In this case something like `process.runtime.dotnet.assembly_count` (using an `_` rather than `-` since I think that's more commonly what I've seen in other semantic conventions).

The spec has some general guidance here for runtime metrics: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/runtime-environment-metrics.md#runtime-environment-specific-metrics---processruntimeenvironment

Reply:
Thank you. Names are now prefixed.